tests/compilers/typed_functions.rs

```diff
@@ -244,11 +244,11 @@ fn static_host_function_without_env(config: crate::Config) -> anyhow::Result<()>
     let mut store = config.store();
 
     fn f(a: i32, b: i64, c: f32, d: f64) -> (f64, f32, i64, i32) {
-        (d * 4.0, c * 3.0, b * 2, a * 1)
+        (d * 4.0, c * 3.0, b * 2, a)
     }
```

Comment:
Not a fan of the "* 1" removal, as it was here for clarity. Still works, but, meh

Reply:
Yeah, it looks like clippy warns you about redundant multiplies so `clippy --fix` will "help" out by removing them :disappointed: